server/src/controllers/forests/firewood-permits.es6

```diff
@@ -0,0 +1,189 @@
+ * @param {Object} res - http response
+ */
+firewoodPermits.create = async (req, res) => {
+  util.logControllerAction(req, 'firewoodPermits.create', req.body);
+
+  const application = req.body;
+  const query = { where: { id: application.forestId } };
+
+  try {
+    const forest = await forestsDb.fsForests.findOne(query);
+
+    if (!forest) {
+      return res.status(400).send();
+    }
+
+    const permitResponse = await firewoodPermitService.createPermitTransaction(application, forest);
+    res.status(200).json(permitResponse);
+  } catch (error) {
+    util.handleErrorResponse(error, res, 'createPermit#end');
+  }
+};
+
+/**
+ * @function getOnePermit - API function to get a permit.
+ * @param {Object} req - http request
+ * @param {Object} res - http response
+ */
+firewoodPermits.getOnePermit = async (req, res) => {
+  util.logControllerAction(req, 'firewoodPermits.getOnePermit', { id: req.params.id });
+
+  const query = {
+    where: { permitId: req.params.id },
+    include: [{ model: forestsDb.fsForests }]
+  };
+
+  try {
+    const permit = await firewoodModel.findOne(query);
+
+    if (!permit) {
+      return res.status(404).send();
+    }
+
+    if (permit.status === 'Error') {
+      return res.status(400).json(formatPermitError(permit));
+    }
+
+    return res.status(200).send(firewoodPermitService.permitResult(permit));
+  } catch (error) {
+    util.handleErrorResponse(error, res, 'getOnePermit#end');
+  }
+};
+
+/**
+ * @function printPermit - API function to get permit svg or rules html
+ * @param {Object} req - http request
+ * @param {Object} res - http response
+ */
+// christmasTreePermits.printPermit = (req, res) => {
+//   treesDb.christmasTreesPermits
+//     .findOne({
+//       where: {
+//         permitId: req.params.id
+//       },
+//       include: [
+//         {
+//           model: treesDb.christmasTreesForests
+//         }
+//       ]
+//     })
+//     .then((permit) => {
+//       util.logControllerAction(req, 'christmasTreePermits.printPermit', permit);
+//       if (permit.status === 'Completed') {
+//         if (!firewoodPermitService.checkPermitValid(permit.permitExpireDate)) {
+//           res.status(404).send();
+//         } else if (!req.query.rules || req.query.permit === 'true') {
+//           permitSvgService.generatePermitSvg(permit).then((permitSvg) => {
+//             res.status(200).json({
+//               result: permitSvg
+//             });
+//           });
+//         } else if (req.query.rules === 'true') {
+//           permitSvgService.generateRulesHtml(false, permit).then((rulesHtml) => {
+//             res.status(200).json({
+//               result: rulesHtml
+//             });
+//           });
+//         } else {
+//           res.status(404).send();
+//         }
+//       } else {
+//         res.status(404).send();
+//       }
+//     })
+//     .catch((error) => {
+//       logger.error(error);
+//       res.status(404).send();
+//     });
+// };
+
```

Comment:
If this is all dead code we should remove it

Reply:
same as above, I would keep for visibility into creating the firewood permit. Our code will likely look similar